### ISSUE TITLE
20210312 Github 前端趋势榜

### ISSUE THREAD

**Issue by kujian** [kujian/githubTrending#2324]
# 20210312 Github 前端趋势榜
Github 前端开发日报栏目数据来自[Github热榜](http://news.caibaojian.com.cn/)（我开发的爬虫），每日分享HTML、CSS、JavaScript、Vue等开源项目，为开发者提供动力，点击Star按钮来关注这个项目，点击Watch来收听每日的更新[Github主页](https://github.com/kujian/githubTrending)
## JavaScript
* [LeCoupa/awesome-cheatsheets](https://github.com/LeCoupa/awesome-cheatsheets) （? Awesome cheatsheets for popular programming languages, frameworks and development tools. They include everything you should know in one single file.）
* [qier222/YesPlayMusic](https://github.com/qier222/YesPlayMusic) （高颜值的第三方网易云播放器，支持 Windows / macOS / Linux）
* [leonardomso/33-js-concepts](https://github.com/leonardomso/33-js-concepts) （? 33 concepts every JavaScript developer should know.）
* [ryanmcdermott/clean-code-javascript](https://github.com/ryanmcdermott/clean-code-javascript) （        ? Clean Code concepts adapted for JavaScript      ）
* [facebook/react](https://github.com/facebook/react) （A declarative, efficient, and flexible JavaScript library for building user interfaces.）
* [MrXujiang/h5-Dooring](https://github.com/MrXujiang/h5-Dooring) （H5 Page Maker, H5 Editor. Make H5 as easy as building blocks. | 让H5制作像搭积木一样简单, 致力于提供一套简单方便、专业可靠、无限可能的H5/PC落地页解决方案.）
* [goldbergyoni/nodebestpractices](https://github.com/goldbergyoni/nodebestpractices) （&#x2705; The Node.js best practices list (June 2020)）
* [laurent22/joplin](https://github.com/laurent22/joplin) （Joplin - a note taking and to-do application with synchronization capabilities for Windows, macOS, Linux, Android and iOS.）
* [twbs/bootstrap](https://github.com/twbs/bootstrap) （The most popular HTML, CSS, and JavaScript framework for developing responsive, mobile first projects on the web.      ）
* [strapi/strapi](https://github.com/strapi/strapi) （? Node.js Content Management Framework (headless-CMS) to build powerful API with no effort.）
* [cypress-io/cypress](https://github.com/cypress-io/cypress) （Fast, easy and reliable testing for anything that runs in a browser.）
* [alpinejs/alpine](https://github.com/alpinejs/alpine) （A rugged, minimal framework for composing JavaScript behavior in your markup.）
* [RocketChat/Rocket.Chat](https://github.com/RocketChat/Rocket.Chat) （Have your own Slack like online chat, built with Meteor.）
* [chartjs/Chart.js](https://github.com/chartjs/Chart.js) （        Simple HTML5 Charts using the &lt;canvas&gt; tag      ）
* [2368693074/49shouting](https://github.com/2368693074/49shouting) （联通挂机任务积分脚本）
* [atom/atom](https://github.com/atom/atom) （        The hackable text editor       ）
* [tmdh/laravel-kit](https://github.com/tmdh/laravel-kit) （A desktop Laravel admin panel app）
* [Leaflet/Leaflet](https://github.com/Leaflet/Leaflet) （? JavaScript library for mobile-friendly interactive maps）
* [alvarotrigo/fullPage.js](https://github.com/alvarotrigo/fullPage.js) （fullPage plugin by Alvaro Trigo. Create full screen pages fast and simple）
* [Working-Title-MSFS-Mods/fspackages](https://github.com/Working-Title-MSFS-Mods/fspackages) （Working Title FS Packages）
* [ampproject/amphtml](https://github.com/ampproject/amphtml) （The AMP web component framework.）
* [carbon-design-system/carbon](https://github.com/carbon-design-system/carbon) （A design system built by IBM）
* [cure53/DOMPurify](https://github.com/cure53/DOMPurify) （DOMPurify - a DOM-only, super-fast, uber-tolerant XSS sanitizer for HTML, MathML and SVG. DOMPurify works with a secure default, but offers a lot of configurability and hooks. Demo:）
* [praveenscience/Internship-LMS-FrontEnd](https://github.com/praveenscience/Internship-LMS-FrontEnd) （We are in the mission of solving a crisis for a lot of small and medium enterprises, who are trying to recruit interns for their projects. This platform aims to onboard new interns, who have applied to the companies and help them understand the process and make the transition smooth.）
* [ether/etherpad-lite](https://github.com/ether/etherpad-lite) （Etherpad: real-time collaborative document editor）

## Vue
* [PanJiaChen/vue-element-admin](https://github.com/PanJiaChen/vue-element-admin) （        vue2 admin / a management system template      ）
* [hoppscotch/hoppscotch](https://github.com/hoppscotch/hoppscotch) （&#x1f47d; A free, fast and beautiful API request builder used by 80k+ developers. https://hoppscotch.io）
* [lyswhut/lx-music-desktop](https://github.com/lyswhut/lx-music-desktop) （一个基于 electron 的音乐软件）
* [meteyou/mainsail](https://github.com/meteyou/mainsail) （A Klipper webinterface made by VoronDesign）
* [chuzhixin/vue-admin-beautiful](https://github.com/chuzhixin/vue-admin-beautiful) （vue-admin-beautiful是一款绝佳的前端开发框架（基于vue/cli 4 最新版，同时支持电脑，手机，平板）,长期更新维护,感谢您的star,我一直在努力 Vue admin beautiful is an excellent front-end development framework (supporting computers, mobile phones and tablets at the same time), long-term update and maintenance, thank you for your star, I have been working hard）
* [woai3c/visual-drag-demo](https://github.com/woai3c/visual-drag-demo) （可视化拖拽组件库 DEMO）
* [salomonelli/best-resume-ever](https://github.com/salomonelli/best-resume-ever) （? ? Build fast ? and easy multiple beautiful resumes and create your best CV ever! Made with Vue and LESS.）
* [vueComponent/ant-design-vue-pro](https://github.com/vueComponent/ant-design-vue-pro) （An enterprise-class UI components based on Ant Design and Vue. &#x1f41c;）
* [weilanwl/ColorUI](https://github.com/weilanwl/ColorUI) （鲜亮的高饱和色彩，专注视觉的小程序组件库）
* [ElemeFE/element](https://github.com/ElemeFE/element) （        A Vue.js 2.0 UI Toolkit for Web      ）
* [cuiocean/ZY-Player-APP](https://github.com/cuiocean/ZY-Player-APP) （&#x25b6;&#xfe0f; 跨平台移动端视频资源播放器. 简洁免费. &#x1f39e; ZY Player 移动端 APP，基于 Uni-app 开发.）
* [directus/directus](https://github.com/directus/directus) （Open-Source Data Platform — Directus wraps your new or existing SQL database with a realtime GraphQL+REST API for developers, and an intuitive admin app for non-technical users. &#x1f430;）
* [Heeexy/SpringBoot-Shiro-Vue](https://github.com/Heeexy/SpringBoot-Shiro-Vue) （提供一套基于Spring Boot-Shiro-Vue的权限管理思路.前后端都加以控制,做到按钮/接口级别的权限）
* [hzeyuan/visit-vault](https://github.com/hzeyuan/visit-vault) （egg.js、graphql、typescript、mongo，图片管理）
* [chaitin/xray](https://github.com/chaitin/xray) （一款完善的安全评估工具，支持常见 web 安全问题扫描和自定义 poc | 使用之前务必先阅读文档）
* [ChanIok/JumpyDumpty](https://github.com/ChanIok/JumpyDumpty) （蹦蹦炸弹——原神小工具箱：热键地图，用户查询，圣遗物OCR导出）
* [statping/statping](https://github.com/statping/statping) （Status Page for monitoring your websites and applications with beautiful graphs, analytics, and plugins. Run on any type of environment.）
* [lusaxweb/vuesax-next](https://github.com/lusaxweb/vuesax-next) （Vuesax v4: framework components for Vuejs）
* [xyxiao001/vue-cropper](https://github.com/xyxiao001/vue-cropper) （A simple picture clipping plugin for vue）
* [iview/iview-admin](https://github.com/iview/iview-admin) （Vue 2.0 admin management system template based on iView）
* [sdras/ecommerce-netlify](https://github.com/sdras/ecommerce-netlify) （&#x1f6cd; A JAMstack Ecommerce Site built with Nuxt and Netlify Functions）
* [nuxt/nuxtjs.org](https://github.com/nuxt/nuxtjs.org) （Nuxt.js Documentation Website (Universal Vue.js Application built with Nuxt.js)）
* [spider-hand/Geoguess-Master](https://github.com/spider-hand/Geoguess-Master) （Free geoguess game）
* [Seagate/cortx-management-portal](https://github.com/Seagate/cortx-management-portal) （CORTX Management Portal provides user interface (UI) to facilitate the easy utility of different CORTX components and features. It communicates with CORTX manager through middleware application. The CORTX Manager communicates with different CORTX components and features and gives proper response to CORTX Management Portal.）
* [airyland/vux](https://github.com/airyland/vux) （        Mobile web UI Components based on Vue and WeUI (Vue@2.x)      ）

## HTML
* [docker/awesome-compose](https://github.com/docker/awesome-compose) （Awesome Docker Compose samples）
* [zwdnet/PythonPractice](https://github.com/zwdnet/PythonPractice) （极客时间课程《Python核心技术与实战》课程练习实践。）
* [mdn/content](https://github.com/mdn/content) （The content behind MDN Web Docs）
* [ganlanyuan/tiny-slider](https://github.com/ganlanyuan/tiny-slider) （Vanilla javascript slider for all purposes.）
* [swapnilsparsh/Rescue](https://github.com/swapnilsparsh/Rescue) （Women Safety Website）
* [tailwindlabs/tailwindcss-forms](https://github.com/tailwindlabs/tailwindcss-forms) （）
* [nathanreyes/v-calendar](https://github.com/nathanreyes/v-calendar) （A lightweight, dependency-free plugin for building attributed calendars in Vue.js）
* [airbnb/lottie](https://github.com/airbnb/lottie) （Lottie documentation for http://airbnb.io/lottie）
* [nicehash/NiceHashQuickMiner](https://github.com/nicehash/NiceHashQuickMiner) （Super simple &amp; easy Windows 10 cryptocurrency miner made by NiceHash.）
* [Algo-Phantoms/Algo-Phantoms-Frontend](https://github.com/Algo-Phantoms/Algo-Phantoms-Frontend) （Website of Algo Phantoms, an Open Source organization focuses on Data Structure and Algorithms.）
* [mdn/learning-area](https://github.com/mdn/learning-area) （Github repo for the MDN Learning Area.）
* [An0nUD4Y/blackeye](https://github.com/An0nUD4Y/blackeye) （The ultimate phishing tool with 38 websites available!）
* [0dayCTF/reverse-shell-generator](https://github.com/0dayCTF/reverse-shell-generator) （Hosted Reverse Shell generator with a ton of functionality. -- (Great for CTF's)）
* [nasa/nasa.github.io](https://github.com/nasa/nasa.github.io) （Information about the NASA Github organization）
* [nginxinc/NGINX-Demos](https://github.com/nginxinc/NGINX-Demos) （        NGINX and NGINX Plus demos      ）
* [web-platform-tests/wpt](https://github.com/web-platform-tests/wpt) （Test suites for Web platform specs — including WHATWG, W3C, and others）
* [github/personal-website](https://github.com/github/personal-website) （Code that'll help you kickstart a personal website that showcases your work as a software developer.）
* [lipis/flag-icon-css](https://github.com/lipis/flag-icon-css) （        ? A collection of all country flags in SVG — plus the CSS for easier integration      ）
* [wblakecannon/DataCamp](https://github.com/wblakecannon/DataCamp) （DataCamp data-science courses）
* [twbs/icons](https://github.com/twbs/icons) （Official open source SVG icon library for Bootstrap.）
* [Ylianst/MeshCentral](https://github.com/Ylianst/MeshCentral) （This is a full computer management web site. With MeshCentral, you can run your own web server and it to remotely manage and control computers on a local network or anywhere on the internet. Once you get the server started, will create a mesh (a group of computers) and then download and install a mesh agent on each computer you want to manage. A…）
* [awantika10/Desgen](https://github.com/awantika10/Desgen) （We provide innovative and latest solutions for our customers. We try to bring revolution in the field of design through latest technologies.）
* [SamProf/MatBlazor](https://github.com/SamProf/MatBlazor) （Material Design components for Blazor and Razor Components）
* [marcel-dempers/docker-development-youtube-series](https://github.com/marcel-dempers/docker-development-youtube-series) （）
* [keycloak/keycloak-documentation](https://github.com/keycloak/keycloak-documentation) （）

## CSS
* [tailwindlabs/tailwindcss](https://github.com/tailwindlabs/tailwindcss) （A utility-first CSS framework for rapid UI development.）
* [necolas/normalize.css](https://github.com/necolas/normalize.css) （A collection of HTML element and attribute style-normalizations      ）
* [gradio-app/gradio](https://github.com/gradio-app/gradio) （Create UIs for prototyping your machine learning model in 3 minutes）
* [bradtraversy/50projects50days](https://github.com/bradtraversy/50projects50days) （50+ mini web projects using HTML, CSS &amp; JS）
* [himanshu007-creator/SFC-foundations](https://github.com/himanshu007-creator/SFC-foundations) （SFC bringing change to the society. Drop a &#x2b50;）
* [cobidev/simplefolio](https://github.com/cobidev/simplefolio) （&#x26a1;&#xfe0f; A minimal portfolio template for Developers）
* [thewhiteh4t/seeker](https://github.com/thewhiteh4t/seeker) （Accurately Locate Smartphones using Social Engineering）
* [DSC-JSS-NOIDA/foss-events](https://github.com/DSC-JSS-NOIDA/foss-events) （A simple website with a collection of open-source events happening across the globe. This is a beginner-friendly repository that helps you learn git and contribute to web projects. Happy Hacktober!）
* [tapaswenipathak/Open-Source-Programs](https://github.com/tapaswenipathak/Open-Source-Programs) （A list of open source programs.）
* [FOSS-Cell-GECPKD/pixelvibe](https://github.com/FOSS-Cell-GECPKD/pixelvibe) （This a Pixel-art maker with add-on features.）
* [john-smilga/react-projects](https://github.com/john-smilga/react-projects) （）
* [apachecn/sklearn-doc-zh](https://github.com/apachecn/sklearn-doc-zh) （&#x1f4d6; [译] scikit-learn（sklearn） 中文文档）
* [primefaces/primeng](https://github.com/primefaces/primeng) （        UI Components for Angular      ）
* [akashyap2013/Blooger_Website](https://github.com/akashyap2013/Blooger_Website) （This is the complete blooger website create using html and css）
* [stripe-samples/checkout-one-time-payments](https://github.com/stripe-samples/checkout-one-time-payments) （Use Checkout to quickly collect one-time payments.）
* [sefyudem/Responsive-Login-Form](https://github.com/sefyudem/Responsive-Login-Form) （）
* [Microsoft/vscode-docs](https://github.com/Microsoft/vscode-docs) （Public documentation for Visual Studio Code）
* [goitacademy/javascript-homework](https://github.com/goitacademy/javascript-homework) （）
* [sefyudem/Sliding-Sign-In-Sign-Up-Form](https://github.com/sefyudem/Sliding-Sign-In-Sign-Up-Form) （）
* [nanxiaobei/hugo-paper](https://github.com/nanxiaobei/hugo-paper) （&#x1f329; A simple, clean, flexible Hugo theme）
* [GeekZoneHQ/web](https://github.com/GeekZoneHQ/web) （Software to power the Geek.Zone website and apps）
* [4linux/DevOpsLab-HelloWorld](https://github.com/4linux/DevOpsLab-HelloWorld) （Aplicação criada para exemplificar o Ciclo de uma PipeLine DevOps）
* [maticnetwork/whitepaper](https://github.com/maticnetwork/whitepaper) （Matic whitepaper）
* [learn-co-curriculum/react-information-flow-code-along](https://github.com/learn-co-curriculum/react-information-flow-code-along) （）
* [CodeByZach/pace](https://github.com/CodeByZach/pace) （Automatically add a progress bar to your site. #hubspot-open-source）

## 关于我
 [前端开发博客](http://caibaojian.com.cn/) 站长，[微博](https://weibo.com/kujian) 粉丝18万，[微信公众号](https://open.weixin.qq.com/qr/code?username=caibaojian_com)。
## 推荐
 关注我的微信公众号：[前端开发博客](https://open.weixin.qq.com/qr/code?username=caibaojian_com)，在后台回复以下关键字获取资源：
 1. 回复「1024」，领取前端进阶资料，包含小程序、Nodejs、Git等
 2. 回复「Vue」获取 Vue 精选文章
 3. 回复「面试」获取 面试 精选文章
 4. 回复「JS」获取 JavaScript 精选文章
 5. 回复「CSS」获取 CSS 精选文章
 6. 回复「加群」进入500人前端精英群
 7. 回复「电子书」下载我整理的大量前端资源，含面试、Vue实战项目、CSS和JavaScript电子书等。
 8. 回复「知识点」下载高清JavaScript知识点图谱
